salt/utils/parsers.py

```diff
@@ -2658,13 +2658,6 @@ def _mixin_setup(self):
             default=None,
             help='Set this directory to search for additional states.'
         )
-        self.add_option(
-            '--retcode-passthrough',
-            default=False,
-            action='store_true',
-            help=('Exit with the salt call retcode and not the salt binary '
-                  'retcode.')
-        )
```

Comment:
I am against removing this, until it goes through a deprecation path, because there are tools out there that use --retcode-passthrough, and they will need to be updated.

Reply:
@gtmanfred fixd.